### PR TITLE
Feature/use docusaurus

### LINKED ISSUE #70: Switch from Doctave to Docusaurus
#### Provide Examples

### PR DESCRIPTION
**Pull request by ingeridhellen**
## Why is this pull request needed?

It has been decided to switch from Doctave to Docusaurus, as the latter has more features and is easier to set up.

## What does this pull request change?

Moves current documentation from Doctave to Docusaurus. It does not change the content of the documentation or its structure, other then what is necessary.
https://equinor.github.io/template-fastapi-react/

## Issues related to this change:

Closes #70

### CODE REVIEW

File: documentation/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "documentation",
+  "version": "0.0.0",
```
Comment (sebastianvitterso) [inline documentation/package.json:3]: `0.0.0`? 😳

**Review by sebastianvitterso — APPROVED** [review]
Looks good!

**Comment by eoaksnes** [comment]
To include plantUML files we can use https://github.com/webgears/remarkable-plantuml

File: documentation/docusaurus.config.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+          ],
+        },
+        footer: {
+          style: 'dark',
+          links: [
+            {
+              title: 'Docs',
+              items: [
+                {
+                  label: 'Tutorial',
```
Comment (eoaksnes) [inline documentation/docusaurus.config.js:82]: Rename to Docs :)

**Review by eoaksnes — CHANGES_REQUESTED** [review]
Rename from Tutorial to Docs + use try to use https://github.com/webgears/remarkable-plantuml to show plantUML diagrams.

**Comment by ingeridhellen** [comment]
> 

Don't think it will work: https://github.com/webgears/remarkable-plantuml/issues/1. Besides, the repo seems forgotten, last commit was 4 years ago.

**Review by sebastianvitterso — APPROVED** [review]

**Comment by ingeridhellen** [comment]
> To include plantUML files we can use https://github.com/webgears/remarkable-plantuml

Don't think it will work: https://github.com/webgears/remarkable-plantuml/issues/1. Besides, the repo seems forgotten, last commit was 4 years ago.

**Review by sebastianvitterso — APPROVED** [review]

**Review by eoaksnes — APPROVED** [review]
Nice work!

**Pull request merged**